ta/pkcs11/src/processing.c

```diff
@@ -27,6 +27,40 @@ static enum pkcs11_rc get_ready_session(struct pkcs11_session *session)
 	return PKCS11_CKR_OK;
 }
 
+static enum processing_func func_for_cmd(enum pkcs11_ta_cmd cmd)
+{
+	switch (cmd) {
+	case PKCS11_CMD_ENCRYPT_INIT:
```

Comment:
We don't need this conversion for _XXX_INIT and _ONESHOT, prefer remove them.

Reply:
OK

Reply:
Noticed that following were too early:
```
+       case PKCS11_CMD_DIGEST_UPDATE:
+       case PKCS11_CMD_DIGEST_KEY:
+       case PKCS11_CMD_DIGEST_FINAL:
+               return PKCS11_FUNCTION_DIGEST;
```
Moved them to next commit.

Reply:
Oh... for ONESHOT it is needed as the _INIT setups the stage.